Makefile

```diff
@@ -302,7 +302,7 @@ setup-check: .env .vscode/settings.json
 .venv:
 # target: .venv – Creates a python virtual environment with dev tools (pip, pylint, ...)
 	python3 -m venv .venv
-	.venv/bin/pip3 install --upgrade pip wheel setuptools
+	.venv/bin/pip3 install --upgrade pip~=19.1.1 wheel setuptools
```

Comment:
hmm, confused now there now it's using ensure_pip in the other PR right?
Also shall we fix it? it was a travis issue after all...

Reply:
Good catch , I forgot to undo it